src/server.ts

```diff
@@ -64,6 +65,35 @@ newApp(
       });
     }
 
+    // Monitor all the requests to the server and track all to AppInsights
+    if (process.env.APPINSIGHTS_INSTRUMENTATIONKEY !== undefined) {
+      const appInsightsClientBuilder = new AppInsightsClientBuilder(
+        process.env.APPINSIGHTS_INSTRUMENTATIONKEY
+      );
+      const appInsightsClient = appInsightsClientBuilder.getClient();
+      server.on(
+        "request",
+        (req: http.IncomingMessage, res: http.ServerResponse) => {
+          const requestTimeMs = Date.now();
+          res.on("finish", () => {
+            const responseTimeMs = Date.now();
```

Comment:
nit: `Date.now()` is usually good fit, but if you want to avoid some pitfalls `process.hrtime` is usually a better choice:
https://blog.abelotech.com/posts/measure-execution-time-nodejs-javascript/

Reply:
Using `trackNodeHttpRequest` is no more necessary. Thanks.